src/functions/severa/list-resource-allocations-by-user/index.ts

```diff
@@ -0,0 +1,16 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      httpApi: {
+        method: "get",
+        path: "/severa/users/{severaUserId}/resourceallocations/allocations",
```

Comment:
Also in the spec its severa/userid/resourcealloactions, I think it should be severa/users/userId/resourceallocations. LEave a comment if this needs clarifying

Reply:
While making this change to the spec I would also suggest changing the severa/{severaUserId}/flextime to be severa/users/{severaUserId}/flextime. It might be worth discussing this change with Aleksi also, I suggest it to make the endpoints clearer, more structured and follow typical REST convention